cocos/2d/CCDrawNode.cpp

```diff
@@ -398,6 +398,21 @@ void DrawNode::onDrawGLLine(const Mat4 &transform, uint32_t flags)
         // texcood
         glVertexAttribPointer(GLProgram::VERTEX_ATTRIB_TEX_COORD, 2, GL_FLOAT, GL_FALSE, sizeof(V2F_C4B_T2F), (GLvoid *)offsetof(V2F_C4B_T2F, texCoords));
     }
+
+#if CC_TARGET_PLATFORM == CC_PLATFORM_MAC || CC_TARGET_PLATFORM == CC_PLATFORM_WIN32
```

Comment:
the issue with this code is that we are adding functionality that only works on certain platforms.
for example, if I do `setLineSmooth(true)` on an iOS device, what will happen?
It won't work, but the user won't receive any kind of error.